### PR TITLE
Added build-web command

### PR DESCRIPTION
**Pull request by pierluca**

### CODE REVIEW

**Review by jiabaow — DISMISSED** [review]
🤩

**Review by Xelowak — APPROVED** [review]
Thanks for the addition ! Actually, I think it is always recommended to run "npx expo-optimize" before building the website. Maybe you can create another command in package.json, or add it directly in the build.

**Review by Xelowak — APPROVED** [review]
Perfect, thank you !

File: fe1-web/package.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
+    "prebuild-web": "node_modules/expo-optimize/build/index.js",
+    "build-web": "expo build:web",
```
Comment (pierluca) [inline fe1-web/package.json:9]: @Xelowak @jiabaow 
FYI, "pre" and "post" are prefixes that can be used to execute some operation before or after a given command.
So `npm run build-web` will actually execute:
- prebuild-web
- build-web
- postbuild-web ( which doesn't exist, so it won't be run )

Comment (Xelowak) [reply fe1-web/package.json:9]: Okay, I didn't know that

**Pull request merged**